Dockerfile

```diff
@@ -20,6 +20,7 @@ RUN echo @edge http://nl.alpinelinux.org/alpine/edge/community > /etc/apk/reposi
       nss@edge \
       freetype@edge \
       ttf-freefont@edge \
+      tzdata@edge
```

Comment:
Needs a line continuation (`\`).

I need to fix GitHub actions to run on PRs...